README.adoc

```diff
@@ -60,8 +60,23 @@ include:
 
 |`bind_address`
 |`"127.0.0.1:12220"`
-|Yes
-|Specifies that the server should bind to the given IP address and TCP port.  In general, servers can bind to more than one IP address and port, but this is not (yet?) supported.
+|No
+|Specifies that the server should bind to the given IP address and TCP port.  In general, servers can bind to more than one IP address and port, but this is not (yet?) supported. Defaults to "127.0.0.1:0".
+
+|`request_body_max_bytes`
+|`4096`
```

Comment:
This description isn't right -- it's copy-pasted from the one above.

Reply:
Oh whoops, I forgot to rewrite it. Thank you